nav2_lifecycle_manager/src/lifecycle_manager_client.cpp

```diff
@@ -28,17 +28,23 @@ using nav2_util::geometry_utils::orientationAroundZAxis;
 
 LifecycleManagerClient::LifecycleManagerClient(
```

Comment:
Actually, you should just create a new constructor for `LifecycleManagerClient`. It doesn't make sense for us to have both `name` `ns` and also `node` since the only reason name and namespace are there are to create a node. So you can create another parallel constructor that instead takes in a node

Reply:
No, it is used to create the service client
`manage_service_name_ = name + std::string("/manage_nodes");`
`manager_client_ = node_->create_client<ManageLifecycleNodes>(manage_service_name_);`

Reply:
Only the `ns` is used only to create the node.
So it's ok to create a new constructor with `name` and `parent node`